hetio/permute.py

```diff
@@ -48,8 +82,49 @@ def permute_graph(graph, multiplier=10, seed=0, metaedge_to_excluded=dict(), log
 
 def permute_pair_list(pair_list, directed=False, multiplier=10, excluded_pair_set=set(), seed=0, log=False):
     """
-    If n_perm is not specific, perform 10 times the number of edges of permutations
-    May not work for directed edges
+    Permute edges (of a single type) in a graph according to the XSwap function
+    described in https://doi.org/f3mn58. This method selects two edges and
+    attempts to swap their endpoints. If the swap would result in a valid edge,
+    the swap proceeds. Otherwise, the swap is skipped. The end result is that
+    node degree is preserved, but edges are shuffled, thereby losing their
+    original meaning.
+
+    Parameters
+    ----------
+    pair_list : list of tuples
+        List of edges to permute. Each edge is represented at a (source,
```

Comment:
typo: `edge is represented as a`

Reply:
I'm happy to fix this, but feel free to open a PR, so you can get commit credit for spotting the typo.

Reply:
Refs https://github.com/hetio/hetio/pull/26. Thanks @kkloste